bin/jobsub_fetchlog

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python3 -I
+
+#
+# jobsub_fetchlog -- tool for downloading job output files from condor
+# COPYRIGHT 2022 FERMI NATIONAL ACCELERATOR LABORATORY
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+import os
+import os.path
+import subprocess
+import shutil
+
+import htcondor
+
+#
+# we are in prefix/bin/jobsub_fetchlog, so find our prefix
+#
+PREFIX = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+#
+# find parts we need in package management
+#
+sys.path.append(os.path.join(PREFIX, "lib"))
+
+#
+# import our local parts
+#
+import creds
+from get_parser import StoreGroupinEnvironment
+from condor import Job
+
+
+def get_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-J", "--jobid", help="job/submission ID", required=True)
+    parser.add_argument(
+        "--destdir",
+        "--dest-dir",
+        "--unzipdir",
+        help="Directory to automatically unarchive logs into",
+    )
+    parser.add_argument(
+        "--archive-format",
+        help='format for downloaded archive: "tar" (default, compressed with gzip) or "zip"',
+        default="tar",
+    )
+    parser.add_argument(
+        "-G",
+        "--group",
+        help="Group/Experiment/Subgroup for priorities and accounting",
+        action=StoreGroupinEnvironment,
+        default=os.environ.get("GROUP", None),
+    )
```

Comment:
This is just copied from lib/get_parser,  do we want to abstract out "common" args? And does it make sense to keep all the jobsub_submit-specific args in lib/get_parser?

Reply:
I wrestled with the same question a couple months ago when doing some work on jobsub_submit, and at the time there wasn’t enough motivation to abstract out common args. I think now there is.  Something like having a common parser in get_parser, import it here, and add to it?

Reply:
I agree. Here's what I've identified as the "common" args:

* -G/--group
* --role
* --subgroup (?)
* --verbose

Anything else you can think of? I'll go ahead and add that change to this PR.

Reply:
> This is just copied from lib/get_parser, do we want to abstract out "common" args? And does it make sense to keep all the jobsub_submit-specific args in lib/get_parser?

We could add a defaulted argument to get_parser() to have it return just the common subset, and a second to add the pass-through list that the jobsub/condor wrapper uses...  So it would be `get_parser(just_common=False, add_passthru=False)`

Reply:
Is the set of jobsub/condor args pretty constant when it's not the common set? If so, this sounds good to me. If not, I'd advocate for having the extra args for each jobsub/condor command in those files themselves.

Reply:
I added a `get_base_parser()` function with the general args in a group, which `get_parser()` expands on for submit. It's a bit backwards maybe, but renaming `get_parser()` would impact a number of packages and tests.